app/templates/login.hbs

```diff
@@ -20,7 +20,7 @@
       <div
         class="au-o-grid__item au-u-1-4@small au-u-1-2@medium au-u-text-right@medium"
       >
-        <AcmidmLogin />
+        <Acmidm::Login />
```

Comment:
With this change there is no "Meld u aan button" anymore it seems:
![image](https://user-images.githubusercontent.com/3816305/161219746-c46c7b31-d15c-4596-b1f1-98f5793e5129.png)
Expected:
![image](https://user-images.githubusercontent.com/3816305/161220303-4f7653d0-9a35-4017-a38d-00d33f947ee3.png)